Sources_v3/StreamChat/Database/DTOs/UserDTO.swift

```diff
@@ -18,7 +18,9 @@ class UserDTO: NSManagedObject {
     @NSManaged var userUpdatedAt: Date
     
     @NSManaged var flaggedBy: CurrentUserDTO?
-    
+    @NSManaged var currentUser: CurrentUserDTO?
+    var isCurrentUser: Bool { currentUser != nil }
+
```

Comment:
This is not needed. You can get `CurrentUserDTO` from any context by calling `session.currentUser()`

Reply:
replaced with a fetch-request in 8ac7bb42aac85e9e420e5cfa0450d6536bfbda46